src/types.js

```diff
@@ -326,6 +326,10 @@ export type ClassComponentMetadata = {
 
 export type ReactHint = {| object: ObjectValue, propertyName: string, args: Array<Value> |};
 
+export type ReactComponentTreeConfig = {
+  serverSideRenderOnly: boolean,
```

Comment:
I don't understand how this new config and `serverSideRenderOnly` property is being tested?

Reply:
`serverSideRenderOnly` is more of a TODO right now for my upcoming PR. I added it as `ReactComponentTreeConfig` can't be an empty object and it seemed somewhat logical to break down the actual implementation of `serverSideRenderOnly` from defining it in config.